scripts/cleanup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+cd "${DIR}/.."
+rm -r node_modules/
+rm yarn.lock
+lerna clean -y
+
```

Comment:
What about adding `yarn install` && `yarn build`? I do not see any option to do the cleanup and do not install and build after it. 

Or which were your intentions? Am I missing something?

Reply:
Just to clean source. You are right, for this operation only. Will add.